src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -29,14 +31,34 @@
     private static final long MAX_TIMESTAMP_FUTURE = 2L * 60L * 60L;
     private static final long MAX_TIMESTAMP_FUTURE_MS = MAX_TIMESTAMP_FUTURE * 1_000L;
 
+
+    //*****************fields for solidification thread*********************************//
```

Comment:
I thought we agreed on (w/o enforcement) `////////` :)

Reply:
:-)